pkgs/stdenv/generic/default.nix

```diff
@@ -223,7 +223,7 @@ let
         # include it in the result, it *is* available to nix-env for queries.
         meta = { }
             # If the packager hasn't specified `outputsToInstall`, choose a default,
-            # namely `p.bin or p.out or p`;
+            # namely `lib.getBin p`;
```

Comment:
Actually, no &ndash; there's the *name* of the output and not the output itself. Now I can't see anything better than the current state of the PR (around this point).